### PR TITLE
TestCollect: split git diff lines by `\t`

### PR DESCRIPTION
**Pull request by dorschw**
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-3962

### CODE REVIEW

File: Tests/scripts/collect_tests/collect_tests.py [hunk 1]
```diff
@@ -579,7 +579,7 @@ def _get_changed_files(self) -> tuple[str, ...]:
 
         # diff is formatted as `M  foo.json\n A  bar.py\n ...`, turning it into ('foo.json', 'bar.py', ...).
         for line in diff.splitlines():
-            match len(parts := line.split()):
+            match len(parts := line.split('\t')):
```
Comment (esharf) [inline Tests/scripts/collect_tests/collect_tests.py:582]:
```suggestion
            match len(parts := re.split(r'\s', line):
```
we can use regex here

**Review by esharf — APPROVED** [review]

File: Tests/scripts/collect_tests/collect_tests.py (hunk 1 above)
Comment (dorschw) [reply Tests/scripts/collect_tests/collect_tests.py:582]: i rather keep it readable. 
`\s` is invalid, see the ticket this issue fixes - caused by files with whitespaces in their name.

Comment (esharf) [reply Tests/scripts/collect_tests/collect_tests.py:582]: \t will not work for renamed files.
Regarding the original problem, you can use the max line argument.

**Review by esharf — APPROVED** [review]

File: Tests/scripts/collect_tests/collect_tests.py (hunk 1 above)
Comment (dorschw) [reply Tests/scripts/collect_tests/collect_tests.py:582]: following offline discussion, `\t` it is

**Pull request merged**